src/dist/replication/lib/duplication/test/load_from_private_log_test.cpp

```diff
@@ -200,9 +192,19 @@ class load_from_private_log_test : public duplication_test_base
         std::map<gpid, decree> replay_condition;
         replay_condition[id] = 0; // duplicating
         mutation_log::replay_callback cb = [](int, mutation_ptr &) { return true; };
-        mutation_log_ptr mlog = new mutation_log_private(
-            _replica->dir(), private_log_size_mb, id, _replica.get(), 1024, 512, 10000);
-        EXPECT_EQ(mlog->open(cb, nullptr, replay_condition), ERR_OK);
+        mutation_log_ptr mlog;
+
+        int try_cnt = 0;
+        while (try_cnt < 5) {
```

Comment:
Why add retry logic here?

Reply:
Because Travis sometimes fails to read from disk, which makes the test failed.